mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown

```diff
@@ -170,6 +176,19 @@ group. You can specify only one value. Structure is [documented below](#nested_a
 
 - - -
 
+<a name="nested_standby_policy"></a>The `standby_policy` block supports:
+
+```hcl
+standby_policy {
+  initial_delay_sec = 20
+  mode = "MANUAL"
+}
+```
```

Comment:
Can you please add a new example configuration to the [top](https://github.com/GoogleCloudPlatform/magic-modules/blob/e3152e7cfe0853be7fb10cea5a6c53d3de06b399/mmv1/third_party/terraform/website/docs/r/compute_region_instance_group_manager.html.markdown?plain=1#L94) of this file, instead of adding here?

Reply:
Please check.